prod.Dockerfile

```diff
@@ -12,17 +12,25 @@ COPY . .
 RUN pnpm install -f --offline
 
 
-FROM base_builder as backend
+FROM base_builder as backend_builder
 WORKDIR /usr/src/app/packages/hoppscotch-backend
-RUN pnpm exec prisma generate
-RUN pnpm run build
+RUN pnpm --filter=hoppscotch-backend deploy /usr/build/backend
+
+FROM node:18-alpine3.16 as backend
+WORKDIR /usr/src/app
+COPY --from=backend_builder /usr/build/backend/dist dist
+COPY --from=backend_builder /usr/build/backend/node_modules node_modules
+COPY --from=backend_builder /usr/build/backend/prisma prisma
+COPY --from=backend_builder /usr/src/app/.env .
+COPY --from=backend_builder /usr/src/app/aio_run.mjs .
+RUN npx prisma generate
 # Remove the env file to avoid backend copying it in and using it
-RUN rm "../../.env"
+RUN rm ".env"
 ENV PRODUCTION="true"
 ENV PORT=3170
 ENV APP_PORT=${PORT}
 ENV DB_URL=${DATABASE_URL}
-CMD ["pnpm", "run", "start:prod"]
+CMD ["node", "dist/main.js"]
 EXPOSE 3170
```

Comment:
Needs to be changed to accommodate changed introduced in subpath.

Reply:
updated as of latest commit